### PR TITLE
Create Day 1 Assessment Test

### PR DESCRIPTION
**Pull request by CoderShailputri**
My answers are mentioned

### CODE REVIEW

File: 00-Python Object and Data Structure Basics/Day 1 Assessment Test [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+Write an equation that uses multiplication, division, an exponent, addition, and subtraction that is equal to 100.25.
+
+Hint: This is just to test your memory of the basic arithmetic commands, work backwards from 100.25
+
+(2**2)*(50-25)+(1/4) 
+
+Answer these 3 questions without typing code. Then type code to check your answer.
+
+What is the value of the expression 4 * (6 + 5)
+44
```
Comment (CoderShailputri) [inline 00-Python Object and Data Structure Basics/Day 1 Assessment Test:28]: Correct

File: 00-Python Object and Data Structure Basics/Day 1 Assessment Test [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+What is the value of the expression 4 * 6 + 5 
+29
```
Comment (CoderShailputri) [inline 00-Python Object and Data Structure Basics/Day 1 Assessment Test:30]: Correct

File: 00-Python Object and Data Structure Basics/Day 1 Assessment Test [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+# Grab 'hello'
+d['simple_key']
+
+d = {'k1':{'k2':'hello'}}
+# Grab 'hello'
+d['k1']['k2']
+
+# Getting a little tricker
+d = {'k1':[{'nest_key':['this is deep',['hello']]}]}
+d['k1']['nest_key'][1]
```
Comment (CoderShailputri) [inline 00-Python Object and Data Structure Basics/Day 1 Assessment Test:85]: WRONG!!

Comment (CoderShailputri) [reply 00-Python Object and Data Structure Basics/Day 1 Assessment Test:85]: Correct is d['k1'][0]['nest_key'][1][0]

File: 00-Python Object and Data Structure Basics/Day 1 Assessment Test (hunk 1 above)
Comment (AKSHAY-GIT-H) [reply 00-Python Object and Data Structure Basics/Day 1 Assessment Test:28]: 88

File: 00-Python Object and Data Structure Basics/Day 1 Assessment Test (hunk 2 above)
Comment (AKSHAY-GIT-H) [reply 00-Python Object and Data Structure Basics/Day 1 Assessment Test:30]: 58

**Pull request closed without merging**